src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -833,13 +829,14 @@ MavlinkReceiver::handle_message_optical_flow_rad(mavlink_message_t *msg)
 		device_id.devid_s.address = msg->sysid;
 
 		d.timestamp = f.timestamp;
-		d.min_distance = 0.3f;
-		d.max_distance = 5.0f;
+		d.min_distance = _param_ekf2_min_rng.get();
```

Comment:
Hmm... px4_fmu-v2 complains that it cannot find the param:
```
'EKF2_MIN_RNG' is not a member of 'px4::params'
```

https://github.com/PX4/PX4-Autopilot/runs/4380959036?check_suite_focus=true

Reply:
I guess this happens for boards which don't have any range finder compiled in.

Reply:
I'll push a fix.

Reply:
d4af52f205

Reply:
Ah of course, makes sense. Thanks :sweat_smile: 
Sorry I was still looking into another CI related issue...